digitalearthau/duplicates.py

```diff
@@ -140,26 +141,28 @@ def _write_csv(unique_fields, dicts, stream, append=False):
     )
 
 
-def main():
-    if len(sys.argv) > 1 and sys.argv[1] == '--help':
-        sys.stderr.write(
-            'Usage: {} [collections...]\n\n'
-            'Where collections are among: \n\t{}\n\n'
-            'Or specify --all to check all\n'.format(
-                sys.argv[0], '\n\t'.join(sorted(collections.registered_collection_names())))
-        )
-        sys.exit(1)
+collections.init_nci_collections(None)
+
+
+@click.command('duplicates')
+@global_cli_options
+@click.option('-a', '--all_', is_flag=True)
```

Comment:
what's up with postfix underscores?

Reply:
The argument to that method is `all_` to avoid shadowing the `all` built-in. It probably shouldn't be exposed on the cli, though. The convention comes from pep8: 

> If a function argument's name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.)

Reply:
```python
@click.option('-a', '--all', 'all_', is_flag=True)
```

will give `--all` command line option and will store it into `all_` variable